src/coffee/product-type-generator.coffee

```diff
@@ -196,8 +196,22 @@ class ProductTypeGenerator
     i18n = {}
     languages = @_languages header, _.keys row
     for language in languages
-      if row["#{header}.#{language}"].trim() isnt ''
-        i18n[language] = row["#{header}.#{language}"].trim()
+      # condition was commented out due to an error:
```

Comment:
I do not get this comment?

which error?

Meanwhile, we should not commit commented out code.